examples/react-shadcn/wxt.config.ts

```diff
@@ -0,0 +1,16 @@
+import { defineConfig } from "wxt";
+import tailwindcss from "@tailwindcss/vite";
+import path from "path";
+
+// See https://wxt.dev/api/config.html
+export default defineConfig({
+  modules: ["@wxt-dev/module-react"],
+  vite: () => ({
+    plugins: [tailwindcss()],
+    resolve: {
+      alias: {
+        "@": path.resolve(__dirname, "./"),
+      },
+    },
```

Comment:
This alias already exists, you don't need to add it. I believe you still need it in your TSConfig because shadcn doesn't fully resolve any configs you extend. But WXT adds this for you automatically.